src/jaguar.toit

```diff
@@ -53,10 +53,11 @@ registry_ / ContainerRegistry ::= ContainerRegistry
 main arguments:
   try:
     catch --trace: registry_.start_installed
-    exception := catch --trace: serve arguments
-    logger.error "rebooting due to $(exception)"
-  finally:
-    esp32.deep_sleep (Duration --s=1)
+    serve arguments
+    unreachable
+  finally: | is_exception exception |
+    cause := is_exception ? exception.value : null
+    logger.error "rebooting due to $cause"
```

Comment:
Is there an automatic reboot when leaving main?

Reply:
Added comment.